doc/api_2_0_spec/admin_api_2_0_spec.md

```diff
@@ -13,12 +13,6 @@ Table of Contents
       * [api/v2/stories/update (PUT)](#apiv2storiesupdate-put)
          * [Input Description](#input-description)
          * [Example](#example-1)
-      * [api/v2/stories/cliff](#apiv2storiescliff)
```

Comment:
Could you restore those two API calls in the docs (both the T&C and the actual entry below)? Those API calls (`api/v2/stories/cliff` and `api/v2/stories/nytlabels`) still exist, the only change is that now they hit the underlying `cliff-annotator` and `nytlabels-annotator` instead of reading the annotations from disk.